tsconfig.json

```diff
@@ -15,15 +15,6 @@
 		"strictNullChecks": true,
 		"preserveConstEnums": true
 	},
-	"exclude": [
-		"node_modules",
-		"bin",
-		"examples",
-		"scripts"
-	],
-	"include": [
-		"v*/**/*",
-		"default/**/*",
-		"common/**/*",
-	]
+	"exclude": ["node_modules"],
```

Comment:
```suggestion
```

Unless you expect any of the include patterns to have `node_modules` this does nothing.

From: https://www.typescriptlang.org/tsconfig#exclude

> Important: `exclude ` *only* changes which files are included as a result of the `include` setting. A file specified by exclude can still become part of your codebase due to an import statement in your code, a `types` inclusion, a `/// <reference` directive, or being specified in the `files` list.